src/vs/workbench/parts/search/browser/searchActions.ts

```diff
@@ -717,7 +718,6 @@ function fileMatchToString(fileMatch: FileMatch, maxMatches: number): { text: st
 		.slice(0, maxMatches)
```

Comment:
Do the FileMatches need to be sorted too?

Reply:
I disagree. All matches should show up in the same order that they show up in the search tree.

I think right now they will be sorted by the backend but that's not guaranteed in the future, so we should use the sorter for everything now.

Reply:
To begin with, what is criteria of sorting matches in a file? if you take by line no, then no need of sorting.
SearchTree matches are sorted by filename not by line contents or line number.
I verified SearchTree matches and copy ALL. Both are same to same.

Reply:
They should be sorted by line, which is what the sorter already does. All you have to do is use it the same way as the change you already made.

And the same should be done for folders.

> I verified SearchTree matches and copy ALL. Both are same to same with mychange.

That might happen but it's not guaranteed to happen every time, so we should enforce the sort order here.